apollo-router/src/axum_factory/axum_http_server_factory.rs

```diff
@@ -468,6 +468,7 @@ async fn handle_graphql(
         Ok(response) => {
             tracing::info!(counter.apollo_router_session_count_active = -1,);
             let (mut parts, body) = response.response.into_parts();
+            //println!("response parts before compression: {parts:?}");
```

Comment:
to delete

Reply:
fixed in [5bd4ab7](https://github.com/apollographql/router/pull/3067/commits/5bd4ab722d8169bd32ec6cd56948ef1cedb34207)